File: c++/crescente.cpp
#include <iostream>
#include <iomanip>
#include <string>
#include <climits>
#include <cmath>

using namespace std;
int main() {
    int x, y;

    cout << "Digite dois numeros: " << endl;
    cin >> x;
    cin >> y;
    do
    {
        if (x < y)
        {
            cout << "CRESCENTE!" << endl; 
        }
        else
        {
            cout << "DECRESCENTE!" << endl;
        }

        cout << "Digite outros dois numeros: " << endl;
        cin >> x;
        cin >> y;
        

    } while (x != y && y != x);
    

    return 0;
}

File: python/crescente.py
x: int
y: int

print("Digite dois numeros:")
x = int(input())
y = int(input())

while x != y and y != x:
    if x < y:
        print("CRESCENTE!")
    else:
        print("DECRESCENTE!")

    print("Digite outros dois numeros:")
    x = int(input())
    y = int(input())